packages/ckan/.editorconfig

```diff
@@ -0,0 +1,5 @@
+[*.{js,jsx,ts,tsx,vue}]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Looks like a sane config to put in root

Reply:
There is already one at the root, so I will remove this one.